base/compiler/ssair/verify.jl

```diff
@@ -71,6 +71,14 @@ function verify_ir(ir::IRCode)
     for (bb, idx) in bbidxiter(ir)
         stmt = ir.stmts[idx]
         stmt === nothing && continue
+        foreachssa(stmt) do ssa
+            x = ir.stmts[ssa.id]
+            if x isa GotoIfNot || x isa GotoNode || x isa ReturnNode ||
+               (x isa Expr && (x.head == :leave || x.head == :gc_preserve_end))
+                @verify_error "Non value statement $(ssa.id) (defined as $(ir.stmts[ssa.id])) used as a value in statement $idx: $stmt"
+                error()
```

Comment:
@Keno, do you still have an issue open for fixing these post-alpha issues with the new optimizer (in this case, specifically referring to the use of `error()` being against code formatting guidelines for Base).

Also, when did the "Missing line number information" check below get disabled? If we aren't careful to track and set this on every line, the LLVM verifier will `abort` the program.